.github/workflows/load-test.yaml

```diff
@@ -0,0 +1,103 @@
+        uses: docker/setup-buildx-action@v2
+
+      - name: Cache Docker layers
+        uses: actions/cache@v3
+        with:
+          path: /tmp/.buildx-cache
+          key: ${{ runner.os }}-buildx-${{ github.event.inputs.ref }}
+          restore-keys: |
+            ${{ runner.os }}-buildx-
+
+      - name: Set AWS Credentials
+        uses: aws-actions/configure-aws-credentials@v2
+        with:
+          role-to-assume: ${{ secrets.AWS_IAM_ROLE }}
+          aws-region: ${{ secrets.AWS_REGION }}
+
+      - name: Login to AWS ECR
+        uses: docker/login-action@v2
+        with:
+          registry: ${{ secrets.OPENFGA_REGISTRY }}
+
+      - name: Build, tag, and push image to AWS ECR
+        timeout-minutes: 10
+        uses: docker/build-push-action@v3
+        with:
+          context: .
+          push: true
+          tags: ${{ secrets.OPENFGA_ECR_URI }}:${{ steps.vars.outputs.sha_short }}
+          cache-from: type=local,src=/tmp/.buildx-cache
+          cache-to: type=local,dest=/tmp/.buildx-cache-new,mode=max
+
+      - name: helm setup
+        uses: koslib/helm-eks-action@master
+        with:
+          command: |
+            aws eks update-kubeconfig --region ${{ secrets.AWS_REGION }} --name ${{ secrets.OPENFGA_EKS_CLUSTER }} --role-arn ${{ secrets.AWS_IAM_ROLE }} --kubeconfig ./kubeconfig
+            echo 'KUBE_CONFIG_DATA<<EOF' >> $GITHUB_ENV
+            echo $(cat ./kubeconfig | base64) >> $GITHUB_ENV
+            echo 'EOF' >> $GITHUB_ENV
+            kubectl config set-context arn:aws:eks:${{ secrets.AWS_REGION }}:${{ secrets.AWS_ACCOUNT }}:cluster/${{ secrets.OPENFGA_EKS_CLUSTER }}
+            kubectl config get-contexts
+# RUN MANUALLY:
+#            eksctl create iamidentitymapping \
+#              --cluster ${{ secrets.OPENFGA_EKS_CLUSTER }} \
+#              --region ${{ secrets.AWS_REGION }} \
+#              --arn ${{ secrets.AWS_IAM_ROLE }} \
+#              --group system:masters \
+#              --no-duplicate-arns \
+#              --username github-action
+
+      - name: helm deploy
+        uses: koslib/helm-eks-action@master
+        env:
+          KUBE_CONFIG_DATA: ${{ env.KUBE_CONFIG_DATA }}
+        with:
+          command: |
+            helm repo add openfga https://openfga.github.io/helm-charts
+            helm repo update
+            helm install openfgabench-${{ steps.vars.outputs.sha_short }} openfga/openfga-benchmark \
+              --set replicaCount=1 \
+              --set image.repository=${{ secrets.OPENFGA_ECR_URI }} \
+              --set image.tag=openfgabench-${{ steps.vars.outputs.sha_short }} \
+              --set datastore.engine=postgres \
+              --set datastore.uri="postgres://postgres:password@openfga-postgresql.default.svc.cluster.local:5432/postgres?sslmode=disable" \
+              --set postgres.enabled=true \
+              --set postgresql.auth.postgresPassword=password \
+              --set postgresql.auth.database=postgres
```

Comment:
1. Do we want to specify the M/N/K param?  Otherwise, by default, M, N and K will be 1.
2. Do we want to specify the CPU and memory for the postgres?
3. Which use case we want to run for?  I think by default it is use case 1.

Reply:
We need the CPU and memory parameters for Postgres to get consistent benchmark runs. For our test runs we've been running with

```
  --set openfga.postgresql.primary.resources.requests.memory=2Gi \
  --set openfga.postgresql.primary.resources.requests.cpu=1.0
```

We also need to set the CPU and memory for the OpenFGA deployment itself. We've been running with:

```
  --set openfga.resources.requests.memory=1Gi \
  --set openfga.resources.requests.cpu=500m
```

Reply:
> 1. Do we want to specify the M/N/K param? Otherwise, by default, M, N and K will be 1.
> 3. Which use case we want to run for? I think by default it is use case 1.

Added input parameters for all 👍